README.md

```diff
@@ -0,0 +1,51 @@
+# Budget Generator
+
+A small web app for making budgets
+
+## Description
+
+This app creates simple budgets for the user and allows the budgets to be saved or downloaded. At the moment, there
+
+is no ability to log in, but that feature is planned to be added in the future.

```

Comment:
@Edison-Stuart I think perhaps these line breaks are a copy-paste artifact; in any case, let's make sure they get cleaned up before we merge this PR.

Reply:
Fixed all line breaks, also caught some spelling mistakes as well as punctuation.